cat_emails/gmail_fetcher.py

```
@@ -1,32 +1,50 @@
-import argparse
-import ell
-import openai
+from typing import Tuple
```

Ruff (0.8.0)

`typing.Tuple` imported but unused

Remove unused import: `typing.Tuple`

(F401)